.buildkite/scripts/test_e2e.sh

```diff
@@ -118,12 +118,12 @@ scenario_basic() {
     # Run the network.
     echo "Starting the test network."
     ${OASIS_NET_RUNNER} \
-        --net.node.binary ${OASIS_NODE} \
-        --net.runtime.binary ${RUNTIME_BINARY} \
-        --net.runtime.loader ${OASIS_CORE_RUNTIME_LOADER} \
-        --net.runtime.genesis_state ${RUNTIME_GENESIS} \
-        --net.keymanager.binary ${OASIS_CORE_KM_BINARY} \
-        --net.epochtime_mock \
+        --fixture.default.node.binary ${OASIS_NODE} \
```

Comment:
This will not work until a new Oasis Core release is made, see https://github.com/oasislabs/oasis-runtime/pull/993#issuecomment-617725391.

Reply:
You can rebase on master which works with latest oasis core release now